### PR TITLE
remove all unused assignments

### PR DESCRIPTION
**Pull request by GlenDC**
the fixes made in this commit were detected with a tool called [ineffassign](https://github.com/gordonklaus/ineffassign),
it's not my tool, but I use for some other projects as well,
and it can be helpful these kind of situations,
as they usually indicate a bug or a potential false positive in a unit test

### CODE REVIEW

File: modules/consensus/difficulty_test.go [hunk 1]
```diff
@@ -285,10 +285,13 @@ func TestStoreBlockTotals(t *testing.T) {
...
-		parentTimestamp := types.Timestamp(0)
-		currentTimestamp := types.Timestamp(0)
-		currentTarget := types.RootTarget
+		var (
+			id              types.BlockID
+			parentTimestamp types.Timestamp
+
+			currentTimestamp = types.Timestamp(0)
+			currentTarget    = types.RootTarget
+		)
```
Comment (lukechampine) [inline modules/consensus/difficulty_test.go]: this is an odd replacement. Looks like maybe it's complaining about `parentTimestamp` being assigned a value, but then that value is never used?

We don't use `var` blocks much, so we should replace this with:
```go
var id types.BlockID
var parentTimestamp, currentTimestamp types.Timestamp
currentTarget := types.RootTarget
```

**Comment by lukechampine** [comment]
thanks! Maybe we'll added `ineffassign` to our CI.

File: modules/consensus/difficulty_test.go (hunk 1 above)
Comment (GlenDC) [reply modules/consensus/difficulty_test.go]: yeah that could work as well, what's the reason for not using var blocks? To keep diffs as small as possible?

**Comment by GlenDC** [comment]
> thanks! Maybe we'll added ineffassign to our CI.

Yeah you might want to do that, easy to miss these. The tool doesn't catch all of them though, but it does help catching at least some of them. Which is something.

**Comment by GlenDC** [comment]
Force pushed my commit as a response to your code review @lukechampine.

**Pull request merged**